core/src/testFixtures/java/org/jobrunr/jobs/RecurringJobTestBuilder.java

```diff
@@ -71,7 +76,12 @@ public RecurringJobTestBuilder withJobDetails(JobDetails jobDetails) {
     }
 
     public RecurringJobTestBuilder withCronExpression(String cronExpression) {
-        this.cronExpression = CronExpression.create(cronExpression);
+        this.schedule = CronExpression.create(cronExpression);
+        return this;
+    }
+
+    public RecurringJobTestBuilder withIntervalExpression(String intervalExpression) {
+        this.schedule = new Interval(Duration.parse(intervalExpression));
         return this;
     }
```

Comment:
Shouldn't we set `createdAt` here immediately? If it's a `RecurringJob` with an IntervalExpression, to be valid, it needs the `createdAt`. What I would do:

```java
    public RecurringJobTestBuilder withIntervalExpression(String intervalExpression) {
        this.withIntervalExpression(intervalExpression, Instant.now());
        return this;
    }

    public RecurringJobTestBuilder withIntervalExpression(String intervalExpression, Instant createdAt) {
        this.schedule = new Interval(Duration.parse(intervalExpression));
        this.createdAt = createdAt;
        return this;
    }
```